### DISCUSSION TITLE
Sign in failed. Reason: Request signInInitiate failed with message: No instance of has been registered.

### DISCUSSION THREAD

**Discussion by sectapunterx** [community/community#48084] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

So i have this text after loggining in Github profile with Socks proxy:
`Sign in failed. Reason: Request signInInitiate failed with message: No instance of has been registered. Context created at: at new   (C:\snapshot\copilot\dist\agent.js) at C:\snapshot\copilot\dist\agent.js at C:\snapshot\copilot\dist\agent.js at C:\snapshot\copilot\dist\agent.js at C:\snapshot\copilot\dist\agent.js at Object.  (C:\snapshot\copilot\dist\agent.js) at Module._compile (pkg/prelude/bootstrap.js:1930:22) at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10) at Module.load (node:internal/modules/cjs/loader:981:32) at Function.Module._load (node:internal/modules/cjs/loader:822:12) , request id: 4, error code: -32603`

**Comment by sectapunterx** [comment, marked as answer]
UPD: Сopilot worked after changing the Socks proxy to HTTP.

**Reply by kcesc04** [reply]
how to do it

**Reply by sectapunterx** [reply]
you buy a http proxy, in the jet brains ide settings you look for proxy settings, the address is all the numbers and dots before the colon, the port is what's after the colon

**Reply by kcesc04** [reply]
thanks for the answer

**Reply by AlphaMeoww** [reply]
thank you bro,I have solved the problem

**Reply by AyushAgnihotri2025** [reply]
I am facing the same issue with VS Code.

**Comment by phageghost** [comment]
Same error message here but I'm not using a proxy and none is configured in PyCharm . . .

**Reply by sectapunterx** [reply]
you don't use a proxy, but you live in a sanctioned country, so you need to use an individual proxy in an unblocked country

**Reply by phageghost** [reply]
I'm afraid I don't quite understand. I live in the U.S. Github is owned by Microsoft, an American corporation. Which sanctions, exactly, are germane here?

**Reply by sectapunterx** [reply]
I don't know, it worked for me after using a proxy.

**Comment by Taina-007** [comment]
**### github copilot labse some error** : Something went wrong checking access: No instance of has been registered. Context created at: at new t.Context (c:\Users\Luzzhen\.vscode\extensions\github.copilot-labs-0.12.791\dist\extension.js:11:216154) at t.activate (c:\Users\Luzzhen\.vscode\extensions\github.copilot-labs-0.12.791\dist\extension.js:11:199764) at Function.db (d:\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:108:14833) at Function.cb (d:\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:108:14532) at d:\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:108:12550 at f.n (d:\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:94:6818) at f.m (d:\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess.js:94:6781) at f.l (d:\Microsoft VS Code\resources\app\out\vs\workbench\api\node\extensionHostProcess....

**Comment by Pony23333** [comment]
same problem as @Ranbogit , is there a way of using it without settign proxy?
